resources/packs/azure/azure.lr

```diff
@@ -874,4 +874,40 @@ private azure.monitor.diagnosticsetting @defaults("id name") {
   properties dict
   // Diagnostic setting storage account
   storageAccount azure.storage.account
+}
+
+// Azure Authorization
+azure.authorization {
+  // The role definitions for the Azure subscription
+  roleDefinitions() []azure.authorization.roleDefinition
+}
+
+// Azure Role Definition
+azure.authorization.roleDefinition @defaults ("id name scopes") {
+  // the identifier of the role definition
```

Comment:
Hey @preslavgerchev what you have is great. The changes I'm suggesting are just to be consistent with other content. I'm slowly making a consistency effort across our docs. :) 

```suggestion
  // ID of the role definition
```